src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -97,25 +181,25 @@ public void testAllVersionsTested() throws Exception {
     }
 
     public void testOldIndexes() throws Exception {
+        setupDataLoadingNodes();
+        
         Collections.shuffle(indexes, getRandom());
         for (String index : indexes) {
+            long startTime = System.currentTimeMillis();
             logger.info("Testing old index " + index);
             assertOldIndexWorks(index);
+            logger.info("Done with " + index + ", took " + ((System.currentTimeMillis() - startTime)/1000.0) + " seconds");
```

Comment:
can we log with a "-->" prefix? i.e.,  "--> done ... " . Makes it easier to find & grep.